.github/ISSUE_TEMPLATE/other.md

```diff
@@ -8,4 +8,16 @@ assignees: ''
 
 ---
 
+[//]: # ( Fill out to the best of your ability.  )
+[//]: # ( If an item is not applicable, feel free to leave it blank. )
+[//]: # ( Mark off checkbox items by putting an x in between the [ ] )
```

Comment:
A common mistake I see is people leaving spaces around the `x`, which breaks the auto-checkbox. Maybe an example would help?
```suggestion
[//]: # ( Mark off checkbox items by putting an x in between the [ ] without spaces, i.e. `[x]`)
```

Reply:
Good point, thank you!